internal/controller/controller_suite_test.go

```diff
@@ -0,0 +1,22 @@
+package controller_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"go.uber.org/goleak"
+)
+
+func TestController(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Controller Suite")
+}
+
+var _ = AfterSuite(func() {
+	Eventually(leaks).ShouldNot(HaveOccurred())
```

Comment:
Does this check for go routine leaks after every test?  Or after the entire suite runs?   It would be great if we could test for leaks after each test.  That would likely make it easier for us to determine where the leak was or at least how to reproduce it.

Reply:
This checks after the entire test has run. I can modify it to do it after each!